frontend/src/selectors/partiesSelectors.js

```diff
@@ -28,7 +29,19 @@ export const getPartyRelationshipTypeHash = createSelector(
 
 export const getDropdownInspectors = createSelector(
   [getInspectors],
-  (parties) => parties.map((party) => ({ value: party.party_guid, label: party.name }))
+  (parties) => {
+    const activeInspectors = parties
+      .filter((inspector) => moment(inspector.expiry_date) >= moment())
+      .map((inspector) => ({ value: inspector.party_guid, label: inspector.name }));
+    const inactiveInspectors = parties
+      .filter((inspector) => moment(inspector.expiry_date) < moment())
+      .map((inspector) => ({ value: inspector.party_guid, label: inspector.name }));
```

Comment:
What sort of numbers are we looking at for the number of inspectors being processed here? If it can get into the thousands, we may want to refactor this so that we only iterate twice instead of 4 times

Reply:
I think thats a possibility but not for a very long time but I get what you're saying.